kolibri/core/assets/src/views/k-icon/index.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+
+  <div class="k-icon-wrapper">
+    <ui-icon
+      :ariaLabel="ariaLabel"
+      ref="icon"
+    >
+      <slot></slot>
+    </ui-icon>
+
+    <ui-tooltip
+      v-if="showTooltip"
+      trigger="icon"
+    >
+      {{ ariaLabel }}
+    </ui-tooltip>
+  </div>
+
+</template>
+
+
+<script>
+
+  import uiIcon from 'keen-ui/src/UiIcon';
+  import uiTooltip from 'keen-ui/src/UiTooltip';
+
+  export default {
+    name: 'kIcon',
+    components: {
+      uiIcon,
+      uiTooltip,
+    },
+    props: {
+      ariaLabel: {
```

Comment:
prop naming is a little confusing since the aria label is re-used for the tooltip. maybe:

* `label`
* `showLabelTooltip`?